### DISCUSSION TITLE
When will v0.8.8 be released?

### DISCUSSION THREAD

**Discussion by miguelsousa** [fonttools/fontbakery#3653] [Q&A] [answered]
@felipesanches Can you provide any estimate of when v0.8.8 might be released?

**Comment by felipesanches** [comment, marked as answer]
I've reassigned all remaining issues to other milestones with the intention to release 0.8.8 ASAP.

But there's only one issue I still want to address before cutting a new release: https://github.com/googlefonts/fontbakery/issues/3583#issuecomment-1058025997

**Reply by miguelsousa** [reply]
Good to know, thanks.
Consider including these changes #3655 as well.

**Reply by miguelsousa** [reply]
@felipesanches do you think the release of v0.8.8 will happen this week?

**Reply by felipesanches** [reply]
yes

**Reply by felipesanches** [reply]
done: https://pypi.org/project/fontbakery/0.8.8/

**Reply by miguelsousa** [reply]
Thank you!
